css/base_style.styl

```diff
@@ -24,15 +24,19 @@
     opacity .5
 
   &-day
+    margin 0
```

Comment:
Needed for `button` element because currently we are not importing any `reset.css`